e2e/playwright/testing-constraints.spec.ts

```diff
@@ -524,7 +533,7 @@ part002 = startSketchOn('XZ')
       })
     }
   })
-  test.describe('Test Angle/Length constraint single selection', () => {
+  test.describe('Test Angle constraint single selection', () => {
```

Comment:
I broke out these tests because I didn't change the Angle constraint's behavior, it still opens the old modal. There are some tricky parts of it I want to break out into a smaller PR.